packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -463,3 +463,48 @@ interface ServiceRegistry {
    */
   readonly containerPort?: number;
 }
+
+/**
+ * Options for adding an AWS x-ray sidecar container to a Task Definition
+ */
+export interface TracingOptions {
+  /**
+   * The minimum number of CPU units to reserve for the container.
+   */
+  readonly cpu?: number
+  /**
+   * The hard limit (in MiB) of memory to present to the container.
+   *
+   * If your container attempts to exceed the allocated memory, the container
+   * is terminated.
+   *
+   * At least one of memoryLimitMiB and memoryReservationMiB is required for non-Fargate services.
+   */
+  readonly memoryLimitMiB?: number;
+
+  /**
+   * The soft limit (in MiB) of memory to reserve for the container.
+   *
+   * When system memory is under contention, Docker attempts to keep the
+   * container memory within the limit. If the container requires more memory,
+   * it can consume up to the value specified by the Memory property or all of
+   * the available memory on the container instance—whichever comes first.
+   *
+   * At least one of memoryLimitMiB and memoryReservationMiB is required for non-Fargate services.
+   */
+  readonly memoryReservationMiB?: number;
+
+  /**
+   * Whether or not this container is essential
+   *
+   * @default false
+   */
+  readonly essential?: boolean;
+
+  /**
+   * Whether to create an AWS log driver
```

Comment:
Should read "whether to send logs to CloudWatch Logs" (the log driver is an implementation detail).

Reply:
Addressed